classes/image-qa-module-sigs.bbclass

```diff
@@ -0,0 +1,95 @@
+do_image_qa_module_sigs() {
+	bzImage="${IMAGE_ROOTFS}/boot/bzImage"
```

Comment:
I missed it during the first review, variables could be made `local`, yet it does not look like this is idiomatic in OE recipes.